src/actions/importing/aioseo-posts-importing-action.php

```diff
@@ -316,4 +371,41 @@ public function query( $limit = false, $just_detect = false ) {
 		);
 		// phpcs:enable WordPress.DB.PreparedSQL.InterpolatedNotPrepared
 	}
+
+	/**
+	 * Imports the post's noindex setting.
+	 *
+	 * @param bool $aioseo_robots_settings AIOSEO's set of robot settings for the post.
+	 *
+	 * @return bool|null The value of Yoast's noindex setting for the post.
+	 */
+	public function post_robots_noindex_import( $aioseo_robots_settings ) {
+		// If robot settings defer to default settings, we have null in the is_robots_noindex field.
+		if ( $aioseo_robots_settings['robots_default'] ) {
```

Comment:
can this key not be present?

Reply:
Not with how AIOSEO data is structured currently.

But yeah, this falls under the point of data validation and we should come up with a complete plan of dealing with all that.

Reply:
Added an `isset()` for this